src/plugin/plugin.ts

```diff
@@ -615,20 +623,21 @@ class Purchases {
    * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults 
    * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
    */
-  public static setup(
-    apiKey: string,
-    appUserID?: string | null,
-    observerMode: boolean = false,
-    userDefaultsSuiteName?: string
-  ): void {
+  public static setup({
+    apiKey,
+    appUserID = null,
+    observerMode = false,
+    userDefaultsSuiteName,
+    useAmazon = false
```

Comment:
Ideally we would pass the store here instead of `useAmazon`. I don't have a preference tho, since passing AppStore as store for example would actually not have much effect lol

Reply:
Yeah, I think passing the boolean flag is how we have it in all the hybrids so we probably want to keep it consistent.